contentcuration/contentcuration/frontend/shared/i18n/index.js

```diff
@@ -23,6 +39,70 @@ export function $trWrapper(nameSpace, defaultMessages, formatter, messageId, arg
   return formatter(message, args);
 }
 
+const defaultLocale = defaultLanguage.id;
+
+export const availableLanguages = {
+  [defaultLocale]: defaultLanguage,
+};
+
+export let currentLanguage = defaultLocale;
+
+// Default to ltr
+export let languageDirection = languageDirections.LTR;
+
+export function getLangDir(id) {
+  return (availableLanguages[id] || {}).lang_direction || languageDirections.LTR;
+}
+
+export function isRtl(id) {
+  return getLangDir(id) === languageDirections.RTL;
+}
+
+export const languageDensities = {
+  englishLike: 'english_like',
+  tall: 'tall',
+  dense: 'dense',
+};
+
+export let languageDensity = languageDensities.englishLike;
+
+const languageDensityMapping = {
+  ar: languageDensities.tall,
+  bn: languageDensities.tall,
+  fa: languageDensities.tall,
+  gu: languageDensities.tall,
+  hi: languageDensities.tall,
+  ja: languageDensities.dense,
+  km: languageDensities.tall,
+  kn: languageDensities.tall,
+  ko: languageDensities.dense,
+  lo: languageDensities.tall,
+  ml: languageDensities.tall,
+  mr: languageDensities.tall,
+  my: languageDensities.tall,
+  ne: languageDensities.tall,
+  pa: languageDensities.tall,
+  si: languageDensities.tall,
+  ta: languageDensities.tall,
+  te: languageDensities.tall,
+  th: languageDensities.tall,
+  ur: languageDensities.tall,
+  vi: languageDensities.tall,
+  zh: languageDensities.dense,
+};
```

Comment:
I believe we never ended up using these in Kolibri so this might not be necessary.

(If we do keep it, might be best to store the info in the `lanuage_info` json file)